### PR TITLE
Add go mod support

### PR DESCRIPTION
**Pull request by sj14**
I've just run `go mod init` and `go mod tidy`.

### CODE REVIEW

**Comment by anatol** [comment]
It would be nice to have a proper go modules support in this project

File: go.sum [hunk 1]
```diff
@@ -0,0 +1,7 @@
+github.com/kr/pretty v0.1.0 h1:L/CwN0zerZDmRFUapSPitk6f+Q3+0za1rQkzVuMiMFI=
```
Comment (dvsekhvalnov) [inline go.sum:1]: mm.. when i ran `go mod tidy` i'm getting:
```
github.com/niemeyer/pretty
``` 

instead of `kr/prety`.

**Comment by dvsekhvalnov** [comment]
Absolutely guys, totally don't mind, thanks for MR.

File: go.sum (hunk 1 above)
Comment (sj14) [reply go.sum]: I've updated it. Please have a look.

**Comment by dvsekhvalnov** [comment]
Thanks for contribution !

**Pull request merged**